tests/test_snapwell_main.py

```diff
@@ -50,9 +65,8 @@ def test_missing_config_restart(capsys, tmp_path):
         config_file.write("wellpath_files:\n")
         config_file.write("  - {well_file: 'a.w', date: '2020-1-1'}\n")
 
-    app = swm.SnapwellApp(["snapwell", config_file_path])
     with pytest.raises(SystemExit) as e:
-        app.load_config(app.parse_args())
+        swm.SnapwellApp(["snapwell", config_file_path])
```

Comment:
Originally the idea was not to crash in the constructor, but this is less overhead :)